Modules/Apps/GitHub/CertUtil_Parser.mkape

```diff
@@ -0,0 +1,18 @@
+Description: A Module to parse Certutil activity
+Category: Windows
+Author: DReneau
+Version: 1.0
+Id: 7d18d1ad-13b5-435c-a5f1-063093e39646
+BinaryUrl: https://github.com/AbdulRhmanAlfaifi/CryptnetURLCacheParser/releases/tag/1.1/CryptnetUrlCacheParser.exe
+ExportFormat: csv
+Processors:
+  - Executable: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe

```

Comment:
Is there a reason why we're calling PowerShell and not the binary itself? Seems like an extra layer is added here unnecessarily. Correct me if I'm wrong please.